Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM fedora:rawhide
+MAINTAINER Vadim Rutkovsky <vrutkovs@redhat.com>
+
+ADD . /opt/gnome-news/
+WORKDIR /opt/gnome-news/
+
+# Install dependencies
+RUN dnf install -y libappstream-glib-devel autoconf autoconf-archive automake \
+    intltool gcc glib2-devel make findutils tar xz
+
+# Build
+RUN ./autogen.sh && make
+
+# Install required check tools
+RUN dnf install -y python3-pep8 python3-pyflakes
```

Comment:
why pyflakes? I personally like pylint and we're using it in https://github.com/rpm-software-management/dnf and related projects.